File: Commander/CommanderControl.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Commander
{
    class CommanderControl
    {
        public CommanderControl(CommanderView leftView, CommanderView rightView)
        {
            this.leftView = leftView;
            this.rightView = rightView;
        }

        public void AdaptPath() => Other.ChangePath(focusedView.Path);

        CommanderView Other { get => focusedView == leftView ? rightView : leftView; }

        public void OnFocus(string id) => focusedView = id == "left" ? leftView : rightView;

        readonly CommanderView leftView;
        readonly CommanderView rightView;
        CommanderView focusedView;
    }
}

File: Commander/CommanderControl.fs
namespace Commander

type CommanderControl (leftView: CommanderView, rightView: CommanderView)  =
    let mutable focusedView: CommanderView = leftView
    
    let getOther () =
        if focusedView = leftView then rightView else leftView

    member this.OnFocus (id: string) = 
        focusedView <- if id = "left" then leftView else rightView

    member this.Copy () =
        focusedView.StartCopy <| getOther ()

    member this.CreateFolder () = focusedView.StartCreateFolder ()
    member this.AdaptPath () = 
        (getOther ()).ChangePath focusedView.Path
    member this.Refresh () = focusedView.Refresh ()
    